lib/ansible/modules/network/nxos/nxos_static_route.py

```diff
@@ -105,78 +105,54 @@ def reconcile_candidate(module, candidate, prefix, w):
 
     parents = []
     commands = []
+    yrc = remove_command.replace('no ', '')
     if w['vrf'] == 'default':
-        config = netcfg.get_section(set_command)
-        if config and state == 'absent':
+        netcfg = str(netcfg).split('\n')
+        ncfg = []
+        for line in netcfg:
+            # remove ip route commands of non-default vrfs from
+            # the running config just in case the same commands
+            # exist in default and non-default vrfs
+            if '  ip route' not in line:
```

Comment:
Just to be sure, are these whitespace for sub command `ip route`?

Reply:
yes, that is correct